source/about/cloud-subscriptions.rst

```diff
@@ -73,6 +86,9 @@ How do I cancel my subscription?
 
 To cancel your subscription, please `contact us <https://customers.mattermost.com/cloud/contact-us>`__.
 
+What happens if my payment fails?
```

Comment:
@johndavidlugtu, would this be a good place for the delinquent information? The other option is a KB article but I'm concerned that it isn't as discoverable as a consolidated article.